src/main/resources/network/brightspots/rcv/config_file_documentation.txt

```diff
@@ -111,6 +111,12 @@ Config file must be valid JSON format. Examples can be found in the test_data fo
         how the program should handle an overvote when it encounters one
         value: "alwaysSkipToNextRank" | "exhaustImmediately" | "exhaustIfMultipleContinuing"
 
+      "randomSeed" optional
+        the integer seed for the application's pseudorandom number generator
+        required if tiebreakMode is random, previousRoundCountsThenRandom, or generatePermutation
+        value: [0..2147483647]
```

Comment:
Please also add this upper bound to the two other fields in this file that have "..]".